components/expansion-panel/test/expansion-panel.test.js

```diff
@@ -80,6 +80,35 @@ describe('expansion panel', () => {
 		expect(headerEl.textContent.trim()).to.equal(headerText);
 	});
 
+	it('should have caption text when caption is set', async () => {
+		const captionText = 'Item caption';
+		const [actualElement] = addElement(
+			textToDomToParent(`<${COMPONENT_NAME} caption="${captionText}"></${COMPONENT_NAME}>`)
+		);
+		await waitNextTask();
+		const caption = actualElement.shadowRoot.querySelector('.expansion-panel-header');
+		expect(caption.textContent.trim()).to.equal(captionText);
+	});
+
+	it('should have meat text when meta is set', async () => {
+		const metaText = 'meta-data';
+		const [actualElement] = addElement(
+			textToDomToParent(`<${COMPONENT_NAME} metaData="${metaText}"></${COMPONENT_NAME}>`)
+		);
+		await waitNextTask();
+		const meta = actualElement.shadowRoot.querySelector('.expansion-panel-header');
+		expect(meta.textContent.trim()).to.equal(metaText);
```

Comment:
What happens if we have both `meta` and `text`?

Reply:
See #1126

Reply:
Text should actually be named as label

Reply:
why label?

Reply:
It's the button's label

Reply:
Can easily support the 2 while deprecating text already

Reply:
but the caption is not the button label the `header` is...

Reply:
I think we are talking about something that is not existing. There is no "text" property.
theses are the "text areas" that have in the expansion panel:
header, caption, meta-data
![image](https://user-images.githubusercontent.com/7512792/154431079-45e58b1a-f204-40f8-858a-528ae61bc466.png)
I'm not sure that caption needs to be replaced with label.

Reply:
removed the caption from this branch